src/main/java/com/google/code/gwt/crop/client/GWTCropperPreview.java

```diff
@@ -94,6 +94,8 @@ public GWTCropperPreview(Dimension dimension, int value){
      * @param aspectRatio
      */
     public void init(String imageUrl, int canvasWidth, int canvasHeight, double aspectRatio) {
+        if (embeddedImage != null) remove(embeddedImage);

```

Comment:
In my case I am reuse same cropper preview instance and each new cropper
rerun preview init() for new source image.

2015-08-27 11:31 GMT+03:00 Ilja Hämäläinen <notifications@github.com>:

> In src/main/java/com/google/code/gwt/crop/client/GWTCropperPreview.java
> <https://github.com/w32blaster/gwt-cropper/pull/26#discussion_r38072918>:
>
> > @@ -94,6 +94,8 @@ public GWTCropperPreview(Dimension dimension, int value){
> >       * @param aspectRatio
> >       */
> >      public void init(String imageUrl, int canvasWidth, int canvasHeight, double aspectRatio) {
> > +        if (embeddedImage != null) remove(embeddedImage);
>
> Hey, just a quick question: is it possible the situation when an image for
> preview has been initialized before the Preview Widget itself?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/w32blaster/gwt-cropper/pull/26/files#r38072918>.
>